servers/zms/src/main/java/com/yahoo/athenz/zms/store/file/FileConnection.java

```diff
@@ -525,14 +525,32 @@ public boolean deletePrincipal(String principalName, boolean subDomains) {
         }
         return true;
     }
-    
+
     Role getRoleObject(DomainStruct domain, String roleName) {
         HashMap<String, Role> roles = domain.getRoles();
         if (roles == null) {
             return null;
         }
         return roles.get(roleName);
     }
+
+    Role getRoleObject(DomainStruct domain, String roleName, Boolean pending) {
+
+        Role role = getRoleObject(domain, roleName);
+        if (pending == Boolean.FALSE) {
+            if (role != null && role.getRoleMembers() != null && !role.getRoleMembers().isEmpty()) {
+                Iterator<RoleMember> roleit = role.getRoleMembers().iterator();
+                RoleMember rm;
+                while (roleit.hasNext()) {
+                    rm = roleit.next();
+                    if (rm != null && rm.getActive() != null && rm.getActive() == Boolean.FALSE) {
```

Comment:
we don't really need rm.getActive() != null part since we're checking for m.getActive() == Boolean.FALSE